src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -190,6 +209,9 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
         dataBuilder.allocatedTrack(getAllocatedTrack(caseData.getClaimValue().toPounds(), caseData.getClaimType()));
         dataBuilder.businessProcess(BusinessProcess.ready(CREATE_CLAIM));
 
+        if (caseData.getRespondent1OrgRegistered() == NO) {
+            dataBuilder.respondent1OrganisationPolicy(null);
+        }
```

Comment:
what is this code doing?

Reply:
respondent1OrganisationPolicy is set in casedata in previous page